tests/cams2_83/test_cams2_83_cli_mos.py

```diff
@@ -25,7 +26,103 @@ def test_eval_mos_dummy(
 ):
     assert list(fake_cache_path.glob("*.pkl"))
 
-    options = f"season 2024-03-01 2024-05-12 --data-path {tmp_path} --coldata-path {tmp_path} --name 'Test'"
+    options = f"season 2024-03-01 2024-05-12 --data-path {tmp_path} --coldata-path {tmp_path} --cache {fake_cache_path} --name 'Test'"
     result = runner.invoke(app, options.split())
     assert result.exit_code == 0
     assert "no output available" in caplog.text
+
+
+def test_eval_mos_standard(
+    fake_cache_path: Path,
+    tmp_path,
+    dataDir,
+    caplog,
+    monkeypatch,
+):
+    def do_not_run(
+        self,
+        model_name=None,
+        obs_name=None,
+        var_list=None,
+        update_interface=True,
+        analysis=False,
+    ):
+        assert model_name is None
+        assert obs_name is None
+        assert var_list is None
+        assert analysis is False
+        assert update_interface is True
+
+    monkeypatch.setattr("pyaerocom.scripts.cams2_83.evaluation.CAMS2_83_Processer.run", do_not_run)
+    options = f"day 2024-03-01 2024-03-01 --data-path {tmp_path} --coldata-path {dataDir} --cache {fake_cache_path} --id mos-colocated-data --name 'Test'"
+    result = runner.invoke(app, options.split())
+    assert result.exit_code == 0
+    map_dir = os.path.join(tmp_path, "cams2-83/mos-colocated-data/map")
+    ts_st1 = os.path.join(
+        tmp_path, "cams2-83/mos-colocated-data/ts/AT0ENK1_EEA-NRT-concno2_Surface.json"
+    )
+    ts_st2 = os.path.join(
+        tmp_path, "cams2-83/mos-colocated-data/ts/AT0ILL1_EEA-NRT-concno2_Surface.json"
+    )
+    ts_st3 = os.path.join(
+        tmp_path, "cams2-83/mos-colocated-data/ts/XK0012A_EEA-NRT-concno2_Surface.json"
+    )
+    hm_dir = os.path.join(tmp_path, "cams2-83/mos-colocated-data/hm")
+    scat_dir = os.path.join(tmp_path, "cams2-83/mos-colocated-data/scat")
+    contour_dir = os.path.join(tmp_path, "cams2-83/mos-colocated-data/contour")
+    fc_dir = os.path.join(tmp_path, "cams2-83/mos-colocated-data/forecast")
+    cfg_out = os.path.join(
+        tmp_path, "cams2-83/mos-colocated-data/cfg_cams2-83_mos-colocated-data.json"
+    )
+    assert Path(map_dir).is_dir()
+    assert Path(ts_st1).is_file()
+    assert Path(ts_st2).is_file()
+    assert Path(ts_st3).is_file()
+    assert Path(hm_dir).is_dir()
+    assert Path(scat_dir).is_dir()
+    assert Path(contour_dir).is_dir()
+    assert Path(fc_dir).is_dir()
+    assert Path(cfg_out).is_file()
```

Comment:
I thinks it would be cleaner to assert just after the definition, e.g.
``` python
    map_dir = os.path.join(tmp_path, "cams2-83/mos-colocated-data/map")
    assert Path(map_dir).is_dir()
    ...
```
which can be can benefit by taking advantage that `tmp_path` is a `Path`, like his
``` python
    map_dir = tmp_path / "cams2-83/mos-colocated-data/map"
    assert map_dir.is_dir()
    ...
```